scripts/run-hwdb-sweep.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import yaml
+#import json
+from pathlib import Path
+import tempfile
+import subprocess
+import signal
+
+script_dir = os.path.dirname(__file__)
+
+# args
+
```

Comment:
nit : remove blank line?

Reply:
This matches how I left the other comments like `# core`, etc in other places so IMO this is fine.